walkthroughs/week-5-tdd/project/src/main/java/com/google/sps/FindMeetingQuery.java

```diff
@@ -15,9 +15,85 @@
 package com.google.sps;
 
 import java.util.Collection;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Arrays;
+import java.util.Collections;
+import java.util.HashSet;
+import java.util.Set;
 
 public final class FindMeetingQuery {
+  private static int DAY_LENGTH = 24*60;
```

Comment:
There is a WHOLE_DAY const in the TimeRange so you can instead compare the request.getDuration() to TimeRange.WHOLE_DAY.getDuration()

Reply:
Thanks for pointing this out!